### PR TITLE
[Docs][Connector-V2] Reconstruct the Socket Source and SftpFile connector document

### PR DESCRIPTION
**Pull request by Hisoka-X**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/source/SftpFile.md [hunk 1]
```diff
@@ -210,50 +147,82 @@ schema {
...
-## Example
+The following example demonstrates how to create a data synchronization job that reads data from sftp and prints it on the local client:
 
-```hocon
+```bash
+# Set the basic configuration of the task to be performed
+env {
+  execution.parallelism = 1
+  job.mode = "BATCH"
+}
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/SftpFile.md]: Add a sample title?

File: docs/en/connector-v2/source/Socket.md [hunk 2]
```diff
@@ -15,58 +17,66 @@ Used to read data from Socket.
...
-```hocon
+The following example demonstrates how to create a data synchronization job that reads data from Socket and prints it on the local client:
+
+```bash
+# Set the basic configuration of the task to be performed
 env {
   execution.parallelism = 1
-  job.mode = "STREAMING"
+  job.mode = "BATCH"
 }
```
Comment (zhilinli123) [inline docs/en/connector-v2/source/Socket.md]: Add a sample title?

**Comment by hailin0** [comment]
Please fix the conflict

File: docs/en/connector-v2/source/Socket.md (hunk 2 above)
Comment (Hisoka-X) [reply docs/en/connector-v2/source/Socket.md]: Added in https://github.com/apache/seatunnel/pull/5386/files#diff-8e6fb7c11c0564bfc369d6926c2ddb6cd8d5fa04d2e617d5b34bbcde569ce268R170

**Comment by Hisoka-X** [comment]
> Please fix the conflict

Done. Please take a look again. @zhilinli123 @hailin0

**Review by zhilinli123 — APPROVED** [review]
Good job +1

**Review by hailin0 — DISMISSED** [review]

**Review by hailin0 — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]

**Pull request merged**